package.json

```diff
@@ -46,6 +46,10 @@
     "through2": "^2.0.0",
     "uglify-js": "^2.4.16"
   },
+  "devEngines": {
+    "node": "0.12 || 4.x || 5.x || 6.x || 7.x",
```

Comment:
Though I wouldn't suggest to anyone to develop babel on node <4. I would say for developing people should use latest or LTS.